bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClientImpl.java

```diff
@@ -0,0 +1,221 @@
+package com.bitso.bitgo.v2;
+
+import com.bitso.bitgo.util.HttpHelper;
+import com.bitso.bitgo.util.SerializationUtil;
+import com.bitso.bitgo.v2.entity.ListWalletResponse;
+import com.bitso.bitgo.v2.entity.SendCoinsResponse;
+import com.bitso.bitgo.v2.entity.Wallet;
+import com.bitso.bitgo.v2.entity.WalletTransferResponse;
+import com.fasterxml.jackson.core.type.TypeReference;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.HttpURLConnection;
+import java.util.*;
+
+/**
+ * Implementation of the BitGo client.
+ *
+ * @author Enrique Zamudio
+ * Date: 5/8/17 4:46 PM
+ */
+@Slf4j
+public class BitGoClientImpl implements BitGoClient {
+
+    /**
+     * The URL for "sendmany" endpoint (must include anything that's appended to baseUrl).
+     */
+    private static final String SEND_MANY_URL = "/$COIN/wallet/$WALLET/sendmany";
+    private static final String LIST_WALLETS_URL = "/$COIN/wallet";
+    private static final String GET_WALLET_URL = "/$COIN/wallet/";
+    private static final String CURRENT_USER_PROFILE_URL = "/user/me";
+    private static final String GET_WALLET_TRANSFER_URL = "/$COIN/wallet/$WALLET/transfer/$TRANSFER";
+    private static final String GET_WALLET_TRANSFER_SEQ_URL = "/$COIN/wallet/$WALLET/transfer/sequenceId/$SEQUENCE";
+    private static final String LIST_WALLET_TRANSFER_URL = "/$COIN/wallet/$WALLET/transfer";
+    private static final String UNLOCK_URL = "/user/unlock";
+
+    private static final String V1_LIST_WALLET_TXN_URL = "/wallet/$WALLET/tx";
+    /**
+     * The base URL (host, port, up to api/v1 for example
```

Comment:
This seems like a wrong comment - this appears to be a v2 API

Reply:
Done, checked in